### PR TITLE
SERVER-41757 - Fix some issues with the documentation WRT to building on recent Fedora/Ubuntu

### PR DESCRIPTION
**Pull request by rbanffy**
Did a check of the build from source process on Ubuntu and Fedora containers with recent packages and found that some parts of the building.md file needed small updates.

### CODE REVIEW

**Comment by dhatcher42** [comment]
Hi @rbanffy! Thanks for submitting these changes to us. While we look into validating them, could you please sign our [Contributor Agreement](http://www.mongodb.com/contributor)?

**Comment by rbanffy** [comment]
I already signed it.

**Comment by dhatcher42** [comment]
My apologies, our internal list was out-of-date. I can confirm I see your signature.

File: docs/building.md [hunk 1]
```diff
@@ -9,9 +9,14 @@ To build MongoDB, you will need:
...
 * On Linux and macOS, the libcurl library and header is required. MacOS includes libcurl.
     * Fedora/RHEL - `dnf install libcurl-devel`
-    * Ubuntu/Debian - `apt-get install libcurl-dev`
+    * Ubuntu/Debian - `libcurl-dev` is provided by three packages. Install one of them:
+      * `libcurl4-openssl-dev`
+      * `libcurl4-nss-dev`
+      * `libcurl4-gnutls-dev`
 * Python 3.7.x and Pip modules:
   * See the section "Python Prerequisites" below.
+* About 600 GB of free disk space (for source code and to build all components).
```
Comment (chasinglogic) [inline docs/building.md]: I would make this more clear that this is only for the all target. Something like:

```
* About 13 GB of free disk space for the core binaries (`mongod`, `mongos`, and `mongo`) and about 600 GB for the all target.
```

Comment (rbanffy) [reply docs/building.md]: That sounds perfect. If you make it a suggestion, it'd be great (and preserve authorship).

File: docs/building.md [hunk 2]
```diff
@@ -86,12 +96,12 @@ Debian/Ubuntu
...
 
-    # aptitude install build-essential
-    # aptitude install libboost-filesystem-dev libboost-program-options-dev libboost-system-dev libboost-thread-dev
+    # apt-get install build-essential
+    # apt-get install libboost-filesystem-dev libboost-program-options-dev libboost-system-dev libboost-thread-dev
 
 To run tests as well, you will need PyMongo:
 
-    # aptitude install python-pymongo
+    # apt-get install python-pymongo
```
[the remark this answers is not in the document]
Comment (rbanffy) [reply docs/building.md]: Done.

**Comment by chasinglogic** [comment]
Merged with commit: https://github.com/mongodb/mongo/commit/7d18b12384e215a75a7288bfc4e621e9ea5318c6

Thanks for your help!

**Pull request closed without merging**